### DISCUSSION TITLE
ARK - SPAWN CREATURES IN GAME WITH TASK

### DISCUSSION THREAD

**Discussion by Loops117** [gameserverapp/Platform#876] [Suggestions]
Similar to how the Integration mod > Teleport player works, i'd like the same ability with spawning creatures.
I'd like to be able to put the creatures blueprint, level, and coordinates on the server i want the creature to spawn.

**Comment by mvaessen** [comment]
Hi,

This is already possible with a wide variety of mods that are available for free. See an example here: https://github.com/gameserverapp/Platform/discussions/710

There are no plans to add this to tasks, since each game will have different needs and it's very hard to maintain with a large amount of games. RCON commands are your best option and are already possible as we speak.
